frontend/public/components/monitoring.tsx

```diff
@@ -1415,7 +1415,7 @@ const AlertingPage: React.SFC<AlertingPageProps> = ({ match }) => {
         <h1 className="co-m-pane__heading">
           <div className="co-m-pane__name co-resource-item">
             <span className="co-resource-item__resource-name" data-test-id="resource-title">
-              Alerting
+              {match.url === ConfigPath ? 'Alertmanager' : 'Alerting'}
```

Comment:
This will set the title for the `ConfigPath` page, but not for the `YAMLPath` page.

Easiest fix is probably to use a flag like `const isAlertmanager = match.url === ConfigPath || match.url === YAMLPath` and use that here and below.

Reply:
updated. thanks @kyoto